### PR TITLE
Cost report form and small fixes

### LINKED ISSUE #61: Improve demo mobile layout
**Description** 

It would be better to have Flottform element sticky on top of the form

**Links / References**

-

**Tasks / Definition of Done**

- [ ] Change mobile layout

**Further information**

-

### LINKED ISSUE #67: Create new custom demo for "Expense report" use case
**Description** 

We want to show that Flottform can be used in different forms for different use cases.

**Links / References**

-

**Tasks / Definition of Done**

- [ ] Create new form for submitting expense report
- [ ] Use it as a show-case for custom UI

**Further information**

-

### PR DESCRIPTION
**Pull request by TamaraBogantseva**
**Checklist**

- [x] Resolves #67 and Resolves #61 
- [x] Labelled and assigned a reviewer

**Reviewer**

- [ ] I've checked out the code and tested it myself.

**Changes**

- _Create new form_
- _Make Flottform container for default UI sticky_
- _Add upload page for customized default ui form_

https://github.com/user-attachments/assets/3bd90548-8fb7-40ab-a8cb-e55c91c1b74f

### CODE REVIEW

File: flottform/demo/src/routes/expense-report-upload/+page.server.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+			paymentMethod,
+			costCenter,
+			costObject,
+			document: uuid,
+			filenameDoc
+		};
+	}
+};
+
+/* eslint-disable  @typescript-eslint/no-explicit-any */
```
Comment (Narigo) [inline flottform/demo/src/routes/expense-report-upload/+page.server.ts:55]: Avoid disable linter with blocks. Could be possible to use `unknown` here

File: flottform/demo/src/routes/expense-report/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,614 @@
...
+		isFlottformDialogOpened = false;
+	};
+
+	const handleFlottformButtonClick = () => {
+		if (!isEndpointCreated) {
+			createWebRtcChannel();
+		}
+	};
+
+	const clientBase = env.PUBLIC_FLOTTFORM_CLIENT_BASE || 'https://192.168.0.21:5177/custom-ui';
```
Comment (Narigo) [inline flottform/demo/src/routes/expense-report/+page.svelte:48]: `custom-ui`? 🤔

I think we should remove the IP addresses in these parts as we've used them during the prototype / Hackathon phase. Would be better to have the env variables documented

**Review by Narigo — COMMENTED** [review]
I like the new use-case! Would be good to see something like "This is a demo form, your uploads will be deleted after X" as we've done in the other forms previously.

I made a few comments what I'd change though

File: flottform/demo/src/routes/+page.svelte [hunk 3]
```diff
@@ -53,6 +53,11 @@
 				showcases how you can leverage Flottform's functionality while styling your forms exactly as
 				you need, offering complete freedom to match your design preferences."
 		/>
+		<DemoLink
```
[the remark this answers is not in the document]
Comment (TamaraBogantseva) [reply flottform/demo/src/routes/+page.svelte:56]: Should we than change routes to `return-and-complaints-default` and `return-and-complaints-custom`? 🤔

**Review by Narigo — APPROVED** [review]

**Pull request merged**